avalon/nuke/pipeline.py

```diff
@@ -261,68 +286,134 @@ def _install_menu():
     context_menu = menu.addMenu(label)
     context_menu.addCommand("Set Context", contextmanager.show)
 
-    menu.addSeparator()
-    menu.addCommand("Create...", creator.show)
-    menu.addCommand("Load...", loader.show)
-    menu.addCommand("Publish...", publish.show)
-    menu.addCommand("Manage...", sceneinventory.show)
-
     menu.addSeparator()
     menu.addCommand("Work Files...",
                     lambda: workfiles.show(
                         os.environ["AVALON_WORKDIR"]
                     )
                     )
+    menu.addSeparator()
+    menu.addCommand("Create...", creator.show)
+    menu.addCommand(
+        "Load...", command=lambda *args:
+        cbloader.show(use_context=True)
+    )
+    menu.addCommand("Publish...", publish.show)
+    menu.addCommand("Manage...", cbsceneinventory.show)
 
     menu.addSeparator()
-    menu.addCommand("Reset Frame Range", reset_frame_range)
+    menu.addCommand("Reset Frame Range", reset_frame_range_handles)
     menu.addCommand("Reset Resolution", reset_resolution)
 
-    menu.addSeparator()
-    menu.addCommand("Reload Pipeline", reload_pipeline)
+    # add reload pipeline only in debug mode
+    if bool(os.getenv("NUKE_DEBUG")):
+        menu.addSeparator()
+        menu.addCommand("Reload Pipeline", reload_pipeline)
 
 
 def _uninstall_menu():
+    """Uninstalling Avalon menu to Nuke
+    """
     menubar = nuke.menu("Nuke")
     menubar.removeItem(api.Session["AVALON_LABEL"])
 
 
-def reset_frame_range():
-    """Set frame range to current asset"""
+def reset_frame_range_handles():
+    """ Set frame range to current asset
+        Also it will set a Viewer range with
+        displayed handles
+    """
 
     fps = float(api.Session.get("AVALON_FPS", 25))
 
     nuke.root()["fps"].setValue(fps)
-    asset = api.Session["AVALON_ASSET"]
-    asset = io.find_one({"name": asset, "type": "asset"})
+    name = api.Session["AVALON_ASSET"]
+    asset = io.find_one({"name": name, "type": "asset"})
+    asset_data = asset.get("data")
+
+    if not asset_data:
+        msg = "Asset {} don't have set any 'data'".format(name)
```

Comment:
If this ever happens there's something else that's wrong because the Asset schema requires `data` to exist, see [here](https://github.com/getavalon/core/blob/60da0a0494fb3fd930680d5eba55e441a9e999f7/avalon/schema/asset-2.0.json#L16). So if you face(d) this problem make sure wherever you are creating assets that you validate the schema before inserting into the database... because this check should be entirely redundant. :)